mk/config.mk

```diff
@@ -176,6 +176,31 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary.
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
+# so early use is limited to a subset of the TEE Internal Core API (crypto...)
+# To use this feature, set EARLY_TA_PATHS to the paths to one or more TA ELF
```

Comment:
As EARLY_TA_PATHS is a configuration directive that affects the core embedded content, i would rather have it prefixed `CFG_`. Your feeling ?